kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/OnboardingForm.vue

```diff
@@ -109,4 +109,9 @@
     margin-top: 24px;
   }
 
+  /deep/ .truncate-text {
```

Comment:
I am guessing that a text truncator is being inserted into one of the slots here, and that's what this `/deep/` selector is targeting? Might we get better results using the new `TextTruncatorCSS` instead?

Reply:
It's in the KRadioButton component - I'm not sure why as it seems a radio button label is better off wrapping but I'm sure there's something I'm missing. I think a follow-up issue in KDS can help address better